src/views/examples/Login.jsx

```diff
@@ -17,6 +17,22 @@ import {
 } from "reactstrap";
 
 class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      type: 'password'
+    }
+    this.showHide = this.showHide.bind(this);
```

Comment:
@iguit0 , if you use arrow functions you don't need to bind the method to the class

Reply:
You mean in line 65 I can just:
`onClick={() => this.showHide}` 
and that's it?

Reply:
Nop. This way: `functionName = () => {//...}`